libs/src/services/ethContracts/index.js

```diff
@@ -237,9 +274,21 @@ class EthContracts {
       var randomValidSPEndpoint = highestFoundSPVersionEndpoints[Math.floor(Math.random() * highestFoundSPVersionEndpoints.length)]
       selectedServiceProvider = randomValidSPEndpoint
     } catch (err) {
-      console.error(`All discovery providers failed for latest ${this.expectedServiceVersions[spType]}`)
       console.error(err)
-      selectedServiceProvider = null
+      console.warn(`All discovery providers failed for latest ${this.expectedServiceVersions[spType]} with healthy block numbers`)
```

Comment:
Id want to replace 'all discovery providers' with the spType function argument, just for clarity